src/main.rs

```diff
@@ -42,6 +43,7 @@ enum Dependency {
     },
 }
 
+/// This type represents the user's configuration of the program
```

Comment:
```suggestion
```

Types name `Config` always represent a configuration of some sort

Reply:
Okey, since this is linked to the previous comment, i will just remove those lines